### ISSUE TITLE
Release tags are not signed

### ISSUE THREAD

**Issue by abstractj** [keycloak-rel/keycloak-rel#48]
In order to secure the supply, chain and bring more confidence to our releases, we should sign all the release tags. It should be possible to accomplish this with Git, but I'm not familiar with the current implementation of our workflows.

**Comment by stianst** [comment]
I agree this is something we should do, and at the same time we can also improve on GPG keys used for Maven artifacts.

Not sure how/what we need to do here though, so we'll need to come up with some plan.
